chapters/ch08.md

```diff
@@ -2,14 +2,20 @@
 
 ->
 
+# Thiết lập một phép đo đơn trị để nhóm của bạn tối ưu
+
 Classification accuracy is an example of a ​**single-number evaluation metric**​: You run your classifier on the dev set (or test set), and get back a single number about what fraction of examples it classified correctly. According to this metric, if classifier A obtains 97% accuracy, and classifier B obtains 90% accuracy, then we judge classifier A to be superior.
 
 -> 
 
+Độ chính xác trong phân loại là một ví dụ của một **phép đo đơn trị**. Khi chạy bộ phân loại trên một tập phát triển (hoặc tập kiểm tra), rồi nhận về một số thể hiện tỉ lệ mẫu được phân loại chính xác trên tổng số mẫu, con số này được định nghĩa là độ chính xác. Nếu bộ phân loại A thu được 97% độ chính xác, và bộ phân loại B thu được 90% độ chính xác, bộ phân loại A sẽ được đánh giá cao hơn theo phép đo này.
```

Comment:
- không có từ *một*
- số --> chỉ số? 
>Khi chạy bộ phân loại trên ~~một~~ tập phát triển (hoặc tập kiểm tra), rồi nhận về một **_chỉ_** số thể hiện tỉ lệ mẫu được phân loại chính xác trên tổng số mẫu, ~~con số này được định nghĩa~~ hay còn gọi là độ chính xác.

Reply:
E chuyển qua như thế này ổn không ạ? Hơi khác dịch word to word nhưng mà cũng vẫn giữ được ý chính

Khi chạy bộ phân loại trên một tập phát triển (hoặc tập kiểm tra), độ chính xác là chỉ số thể hiện tỉ lệ mẫu được phân loại chính xác trên tổng số mẫu trong tập.

Reply:
- độ chính xác "được tính bằng" ... trong tập "đó"

Reply:
* " Khi chạy bộ phân loại trên một tập phát triển" -> "Khi **bạn** chay bộ phân loại trên ~~một~~ tập phát triển"
*  "Nếu bộ phân loại A thu được 97% độ chính xác" -> "Nếu bộ phân loại A có độ chính xác là 97%" thì có lẽ nó đọc mượt hơn